src/views/Networking.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="networking-page"
+    >
+      <input
+        v-model="room"
+        :placeholder="$t('Networking.Placeholder')"
+      >
+      <button @click="newRoom(room)">
```

Comment:
```suggestion
      <button @click="newRoom()">
```

Reply:
Also, can it be just a link? Like, 
```suggestion
      <RouterLink :to="{ name: 'conference', params: { room } }">
```

Reply:
I guess the link is better